frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -1,105 +1,141 @@
 # Contributing Data
 
-CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
+CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! Please see below for If you have single-cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
```

Comment:
I prefer the original. This update has some editorial issues ("for If"). A run-on "Please ..., please ..."

The last sentence also is a bit of duplicate of the next section

_To confirm we can support your submission, please [contact us] with a description of the data that you'd like to contribute_

Could you combine:

1. Read the requirements
2. Contact us
3. We approve/reject

in Contributing Data?

Then the next section is about the process when approved?

Reply:
Agreed this is wonky & redundant with the next section. How about separation in the other direction...

**Contributing Data** is just a statement clarifying that we do take community submissions...
>CELLxGENE supports a rapidly growing single-cell data corpus because of generous contributions from researchers like you!

**~Publishing~ Submission & Publication Process**
1. Review the Data Eligibility criteria to ensure your data complies with these requirements
2. Contact us to confirm...
3. Once confirmed, you send us files...
4. We upload...
5. ...can be revised...
6. The data are made openly available...

Reply:
I like that structure.